homeUi/src/main/kotlin/com/gravatar/app/homeUi/presentation/home/profile/about/AboutEditField.kt

```diff
@@ -26,35 +33,53 @@ internal fun AboutEditField(
     modifier: Modifier = Modifier,
     maxLines: Int = 1,
     description: String? = null,
+    edited: Boolean = false,
 ) {
+    val inputFieldShape = RoundedCornerShape(8.dp)
+    var isFocused by remember { mutableStateOf(false) }
```

Comment:
Kept here intentionally. When we change orientation, the soft keyboard and the focus are gone, so I didn't want to have this state in the VM.